matter_server/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 6.2.1
```

Comment:
What's with the version jump? Are we trying to stay with the server version? I don't think we should do that. We should just follow semantic versioning and remain logical. This add-on is separate from the server.